modules/datadict/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary.
+
+## Scope
+
+The displays (and depending on permissions) manages the data
+dictionary for things already stored in the LORIS `parameter_type`
+table. This table is generally autopopulated and provides a dictionary
+for imaging headers (populated by the imaging pipeline) and behavioural
+instrument (populated by the `lorisform_parser.php` script).
```

Comment:
I would remove "populated by the"

Reply:
I think if I removed it it wouldn't be clear enough from the context what the brackets mean.